PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -82,20 +83,23 @@ struct HfCandidateCreatorToXiPi {
 
   int runNumber;
 
+  Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0); // filter to use only HF selected collisions
+  Filter filterSelectIndexes = (aod::hf_track_index::hfflag == 1);
+  Filter filterSelectTrackIds = (aod::hf_sel_track::isSelProng > 0);
```

Comment:
Do not move filters before type declarations.

Reply:
Ok I will move them back

Reply:
@vkucera I also moved again the filters in trackIndexSkimCreator for the LfCascades struct

Reply:
Thanks